lang-python/fastbencode/autobuild/defines

```diff
@@ -0,0 +1,15 @@
+PKGNAME=fastbencode
+PKGSEC=python
+PKGDEP="python-3"
+BUILDDEP="python-build python-installer wheel setuptools-python3 setuptools-rust llvm cython"
+PKGDES="Fast implementation of bencode"
+
+ABTYPE=python
+NOPYTHON2=1
+
+# ld.lld does not support loongson3 and mips64r6el
+USECLANG=1
+USECLANG__LOONGSON3=0
+NOLTO__LOONGSON3=1
+USECLANG__MIPS64R6EL=0
+NOLTO__MIPS64R6EL=1
```

Comment:
Please drop the `MIPS64R6EL` references, as we no longer ship this architecture.